CDP4JsonSerializer.Tests/PostOperation/CdpPostOperation.cs

```diff
@@ -5,47 +5,46 @@
 //    Author: Sam Gerené, Merlin Bieze, Alex Vorobiev, Naron Phou, Alexander van Delft
 //
 //    This file is part of CDP4-COMET SDK Community Edition
-//
+// 
 //    The CDP4-COMET SDK Community Edition is free software; you can redistribute it and/or
 //    modify it under the terms of the GNU Lesser General Public
 //    License as published by the Free Software Foundation; either
 //    version 3 of the License, or (at your option) any later version.
-//
+// 
 //    The CDP4-COMET SDK Community Edition is distributed in the hope that it will be useful,
 //    but WITHOUT ANY WARRANTY; without even the implied warranty of
 //    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
 //    Lesser General Public License for more details.
-//
+// 
 //    You should have received a copy of the GNU Lesser General Public License
 //    along with this program; if not, write to the Free Software Foundation,
 //    Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 // </copyright>
-// --------------------------------------------------------------------------------------------------------------------
+// -------------------------------------------------------------------------------------------------------------------------------
 
 namespace CDP4JsonSerializer.Tests.Cdp4PostOperation
 {
     using System.Collections.Generic;
+    using System.Text.Json.Serialization;
 
     using CDP4Common;
     using CDP4Common.DTO;
 
-    using Newtonsoft.Json;
-
     /// <summary>
     /// A CdpPostOperation class used for testing purposes
     /// </summary>
     public class CdpPostOperation

```

Comment:
please use the `PostOperation` from the `CDP4DalCommon` library to test

Reply:
Implemented correctly now, inheriting abstract base class PostOperation.

Reply:
Refactored tests using this to use CDP4DalJsonSerializer tat is able to serialize the generic PostOperation correctly.